### PR TITLE
fix(recipe-callbar-ai-popover): fixed popover positioning in the callbar

### PR DESCRIPTION
**Pull request by ramya-dp**
# PR Title

fixed popover positioning in the callbar

## :hammer_and_wrench: Type Of Change

- [X] Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation

## :book: Description

fixed popover positioning in the callbar by moving it up and preventing overlap.

## :bulb: Context
https://dialpad.atlassian.net/browse/DP-69620

## :pencil: Checklist

- [X] I have reviewed my changes
- [ ] I have added tests
- [ ] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :camera: Screenshots / GIFs
Screen Shot 2023-04-14 at 3 05 00 PM

### CODE REVIEW

**Comment by juliodialpad** [comment]
Wouldn't it just be better to add that `d-mb12` class through the `contentClass` prop?

File: recipes/buttons/callbar_button_with_popover/callbar_button_with_popover.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
       :show-close-button="showCloseButton"
       padding="none"
       class="dt-recipe--callbar-button-with-popover--popover-wrapper"
-      :dialog-class="['dt-recipe--callbar-button-with-popover--popover', contentClass]"
+      :dialog-class="['dt-recipe--callbar-button-with-popover--popover d-mb12', contentClass]"
```
Comment (braddialpad) [inline recipes/buttons/callbar_button_with_popover/callbar_button_with_popover.vue]: Like tooltip, you may also use offset on dt-popover, is that what you're looking to do?

Comment (ramya-dp) [reply recipes/buttons/callbar_button_with_popover/callbar_button_with_popover.vue]: Yes! offset is better, updated.

Screen Shot 2023-04-14 at 3 24 21 PM

**Comment by iropolo** [comment]
LGTM with the latest update

**Review by braddialpad — APPROVED** [review]
thanks!

**Pull request merged**